gateway/src/tests.rs

```diff
@@ -170,3 +254,440 @@ fn validate_proof_not_current_epoch() {
+        let (msg, _raw_proof) = proof::proof_tests::decode_input(&input).expect("Input should decode");
+
+        let res = ethabi::decode(
+            &[
+                // chainId
+                ParamType::Uint(256),
+                // commandIds
+                ParamType::Array(Box::new(ParamType::FixedBytes(32))),
+                // commands
+                ParamType::Array(Box::new(ParamType::String)),
+                // calls
+                ParamType::Array(Box::new(ParamType::Bytes)),
+            ],
+            &msg,
+        );
+
+        match res {
+            Ok(params) => {
+                match params.as_slice() {
```

Comment:
We could avoid some indentation here
```suggestion
        match res.expect("failed to decode msg input").as_slice() {
```